### PR TITLE
feat(root): implement Eddsa multisig derivation 

### PR DESCRIPTION
**Pull request by alebusse**
Implemented Eddsa multisig derivation

BREAKING CHANGE: `wallet.getPrv()` is now an async method, `coin.generateKeyPair()`,
`coin.deriveKeyWithSeed()` and `keychains.create()` are deprecated, use their Async version instead.

[WP-1401](https://bitgoinc.atlassian.net/browse/WP-1401)

TICKET: [WP-1401](https://bitgoinc.atlassian.net/browse/WP-1401)

[WP-1401]: https://bitgoinc.atlassian.net/browse/WP-1401?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WP-1401]: https://bitgoinc.atlassian.net/browse/WP-1401?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by OttoAllmendinger — COMMENTED** [review]
flush draft

**Comment by mmcshinsky-bitgo** [comment]
@alebusse BREAKING CHANGE needs to be in the commit footer to be recognized correctly

File: modules/sdk-core/src/account-lib/index.ts [hunk 1]
```diff
@@ -5,5 +5,7 @@ export * from './mpc';
 export * from './util/crypto';
 // Deprecated
 export * as acountLibCrypto from './util/crypto';
+// Deprecated
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/account-lib/index.ts:8]: redundant comment

File: modules/sdk-core/src/account-lib/util/eddsaKeyDeriver.ts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  bigIntFromBufferBE,
+  getDerivationPath,
+} from '../mpc';
+
+export type RootKeys = {
+  prv: string;
+  pub: string;
+};
+
+export class EddsaKeyDeriver {
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/account-lib/util/eddsaKeyDeriver.ts:19]: Needs JS Docs for methods

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]
@alebusse BREAKING CHANGE needs to be in the commit footer to be recognized correctly

**Comment by mmcshinsky-bitgo** [comment]
@alebusse There are modules that are only test that aren't part of the breaking change for the module itself. This PR needs to be broken up into multiple commits so that the breaking change doesn't wrongfully bump incorrect packages.

**Comment by alebusse** [comment]
> @alebusse BREAKING CHANGE needs to be in the commit footer to be recognized correctly

good to know, i didnt manually created the commmit msg, commitizen cli does it for me, i follow the steps.

**Comment by alebusse** [comment]
> @alebusse There are modules that are only test that aren't part of the breaking change for the module itself. This PR needs to be broken up into multiple commits so that the breaking change doesn't wrongfully bump incorrect packages.

Make sense, this is not actually a PR to merge or review, is just a POC for Islam to review and work on the missing part. But i think we wont even be doing this now, im testing something else

**Pull request closed without merging**